content/applications/inventory_and_mrp/maintenance/maintenance_requests.rst

```diff
@@ -0,0 +1,98 @@
+====================
+Maintenance requests
+====================
+
+.. |MO| replace:: :abbr:`MO (Manufacturing Order)`
+
+In order to keep equipment and work centers functioning properly, it is often necessary to perform
+maintenance on them. This can include preventive maintenance, intended to prevent equipment from
+breaking down, or corrective maintenance, which is used to fix equipment that is broken or otherwise
+unusable.
+
+In Odoo *Maintenance*, users can create *maintenance requests* to schedule and track the progress of
+equipment and work center maintenance.
+
+Create maintenance request
+==========================
+
+To create a new maintenance request, navigate to :menuselection:`Maintenance --> Maintenance -->
+Maintenance Requests`, and click :guilabel:`New`.
+
+Begin filling out the form by entering a descriptive title in the :guilabel:`Request` field (e.g.,
+"Drill not working").
+
+In the :guilabel:`For` drop-down menu, select :guilabel:`Equipment` if the maintenance request is
+being created for a piece of equipment, or :guilabel:`Work Center` if it is being created for a work
+center.
+
+Depending on the option selected in the :guilabel:`For` field, the next field is titled either
+:guilabel:`Equipment` or :guilabel:`Work Center`. Using the drop-down menu for either field, select
+a piece of equipment or a work center.
+
```

Comment:
I'm also seeing a 'Worksheet Template' field in the runbot I'm using -- is this present in most dbs or is this runbot a bit "funky"? haha worth a quick double-check either way